docs/src/componentDocs/ScoreCard/playground/PreviewComponent.tsx

```diff
@@ -0,0 +1,210 @@
+import { ScoreCard } from '@brightlayer-ui/react-components/core/ScoreCard';
+import { InfoListItem } from '@brightlayer-ui/react-components/core/InfoListItem';
+import { Hero } from '@brightlayer-ui/react-components/core/Hero';
+import { HeroBanner } from '@brightlayer-ui/react-components/core/HeroBanner';
+import Cloud from '@mui/icons-material/Cloud';
+import Favorite from '@mui/icons-material/Favorite';
+import Humidity from '@brightlayer-ui/icons-mui/Moisture';
+import List from '@mui/material/List';
+import ListAlt from '@mui/icons-material/ListAlt';
+import Mail from '@mui/icons-material/Mail';
+import MoreVert from '@mui/icons-material/MoreVert';
+import Notifications from '@mui/icons-material/Notifications';
+import Search from '@mui/icons-material/Search';
+import Temp from '@brightlayer-ui/icons-mui/Temp';
+import { createProps, getImage, hideDefaultPropsFromSnippet, removeEmptyLines } from '../../../shared/utilities';
+import { PropsType } from '../../../__types__';
+import PreviewComponentWithCode from '../../../shared/PreviewComponentWithCode';
+
+import * as Colors from '@brightlayer-ui/colors';
+
+export const PreviewComponent = (): JSX.Element => {
+    const scoreCardJson = useAppSelector((state: RootState) => state.componentsPropsState.scoreCardComponent);
+
+    const scoreCardProps = createProps(scoreCardJson.props as PropsType[]);
+    const scoreCardOtherProps = createProps(scoreCardJson.otherProps as PropsType[]);
+
+    const toggleDefaultProp = (propName: string, currentValue: any): string =>
+        hideDefaultPropsFromSnippet(scoreCardJson, propName, currentValue, 'props');
+
+    const actionItems = [
+        <Search key={'search'} />,
+        <Mail key={'mail'} />,
+        <Notifications key={'notifications'} />,
+        <Favorite key={'favorite'} />,
+        <Cloud key={'cloud'} />,
+        <MoreVert key={'morevert'} />,
+    ];
+
+    const heroes: JSX.Element[] = [
+        <Hero
+            key={'hero1'}
+            icon={<Temp fontSize={'inherit'} htmlColor={Colors.black[500]} />}
+            label={'Temperature'}
+            iconSize={48}
+            iconBackgroundColor={Colors.white[50]}
+            ChannelValueProps={{ value: 98, units: '°F' }}
+            fontSize={'normal'}
+        />,
+        <Hero
+            key={'hero2'}
+            icon={<Humidity fontSize={'inherit'} htmlColor={Colors.blue[300]} />}
+            label={'Humidity'}
+            iconBackgroundColor={Colors.white[50]}
+            ChannelValueProps={{ value: 54, units: '%' }}
+            iconSize={48}
+            fontSize={'normal'}
+        />,
+    ];
+
+    const showHeroSection = (noOfHeroes: number): string => {
+        switch (noOfHeroes) {
+            case 0:
+                return '';
+            case 1:
+                return `badge={
+            <HeroBanner>
+                <Hero
+                    key={'hero1'}
+                    icon={<Temp fontSize={'inherit'} htmlColor={Colors.black[500]} />}
```

Comment:
let's take the `htmlColor={Colors.black[500]}` out; I don't think this style is necessary